src/riak_dt_pb.erl

```diff
@@ -4145,6 +4145,7 @@ verify_msg(Msg, Opts) ->
     end.
 
 
+-dialyzer({nowarn_function,v_msg_mapfield/3}).
```

Comment:
It's a compilation bug. If you run `protogen` using a "too new" Erlang, `gpb` will insert these.